android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -878,9 +878,8 @@ public void onStreamCreated(PublisherKit publisherKit, Stream stream) {
         ConcurrentHashMap<String, Stream> mSubscriberStreams = sharedState.getSubscriberStreams();
         mSubscriberStreams.put(stream.getStreamId(), stream);
         if (publisherId.length() > 0) {
-            String event = publisherId + ":" + publisherPreface + "onStreamCreated";
             WritableMap streamInfo = EventUtils.prepareJSStreamMap(stream, publisherKit.getSession());
-            sendEventMap(this.getReactApplicationContext(), event, streamInfo);
+            streamInfo.putString("publisherId", publisherId);
             sendEventMap(this.getReactApplicationContext(), "publisherStreamCreated", streamInfo);
```

Comment:
React Native on Android does now appear to like the sequential calls to emit events over the bridge, so now both the OTSubscriber and OTPublisher JS components will receive the one event. The OTPublisher will filter events based on the `publisherId` in the event data (not in the event name).